src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/DefaultRazorProjectChangePublisherTest.cs

```diff
@@ -79,7 +109,8 @@ public async Task ProjectManager_Changed_ProjectRemoved_AfterEnqueuedPublishAsyn
             publisher.ProjectSnapshotManager_Changed(null, args);
 
             // Assert
-            await Task.Delay(publisher.EnqueueDelay * 3).ConfigureAwait(false);
+            var kvp = Assert.Single(publisher._deferredPublishTasks);
```

Comment:
I noticed that there were some tests in this file which weren't testing quite what might be expected of them.